src/Symfony/Component/Yaml/Inline.php

```diff
@@ -684,7 +684,7 @@ public static function evaluateBinaryScalar($scalar)
 
     private static function isBinaryString($value)
     {
-        return !preg_match('//u', $value) || preg_match('/[^\x09-\x0d\x20-\xff]/', $value);
+        return !preg_match('//u', $value) || preg_match('/[^\x00-\x1f\x20-\xff]/', $value);
```

Comment:
in fact i'm not even sure we should use `!!binary` when dumping, most (if not all?) characters are supported in double-quoted anyway.
I'll try to find what the spec says about that.